src/redux/books/books.js

```diff
@@ -0,0 +1,71 @@
+import { without } from 'lodash';
+
+const initBooks = [
+  {
+    id: 'The Hunger Games',
+    title: 'The Hunger Games',
+    genres: 'Action',
+    authors: 'Suzanne Collins',
+    completed: 64,
+    currentChapter: 'Chapter 17',
+  },
+  {
+    id: 'Dune',
+    title: 'Dune',
+    genres: 'Science Fiction',
+    authors: 'Frank Herbert',
+    completed: 8,
+    currentChapter: 'Chapter 3: "A Lesson Learned"',
+  },
+  {
+    id: 'Capital in the Twenty-First Century',
+    title: 'Capital in the Twenty-First Century',
+    genres: 'Economy',
+    authors: 'Suzanne Collins',
+    completed: 0,
+    currentChapter: 'Introduction',
+  },
+
+];
```

Comment:
- [ ]  It is a requirement that you SET the intial STATE with an empty array, kindly fix this, so you should make it like this 
because like this you made empty objects inside the array and this way you specified the number of objects as three but we want it in dynamic way 😊🚀🔥✨

> const initBooks = [ ]